src/test/java/org/smartregister/converters/PhysicalLocationAndStocksConverterTest.java

```diff
@@ -40,6 +48,12 @@ public void testConvertLocationAndStocksToBundleResourceShouldReturnLocationStoc
 		assertTrue(PathEvaluatorLibrary.getInstance()
 				.evaluateBooleanExpression(bundle,
 						"Bundle.entry.resource.ofType(SupplyDelivery).where(id='05934ae338431f28bf6793b24181ea5e').exists()"));
+		assertTrue(PathEvaluatorLibrary.getInstance()
+				.evaluateBooleanExpression(bundle,
+						"Bundle.entry.resource.ofType(SupplyDelivery).identifier.where(system='isPastAccountabilityDate' and value='true').exists()"));
+		assertTrue(PathEvaluatorLibrary.getInstance()
+				.evaluateBooleanExpression(bundle,
+						"Bundle.entry.resource.ofType(SupplyDelivery).identifier.where(system='isPastAccountabilityDate' and value='false').exists()"));
```

Comment:
I find the two added assertions are contradicting. The same identifier having both values `true` and `false`. Does this mean that the code is buggy?

Reply:
Nope there are two stock items one has `accountabilityEndDate` greater than today and the other is in the past.

Reply:
Cool